poppy/transport/pecan/controllers/v1/ssl_certificates.py

```diff
@@ -72,14 +72,20 @@ def post(self):
             helpers.abort_with_message)
     )
     def delete(self, domain_name):
-        # For now we only support 'san' cert type
-        cert_type = pecan.request.GET.get('cert_type', 'san')
-
         certificate_controller = \
             self._driver.manager.ssl_certificate_controller
+
```

Comment:
existing implementation looks good. is it possible to use the existing implementation?